### ISSUE TITLE
Performance bug when creating SparseMatrixCSC

### ISSUE THREAD

**Issue by TomDeWeer** [JuliaLang/julia#34479]
Hi all,

I am trying to solve Ax=B for a sparse A in Julia. I create the A and B matrix in Python and import it into Julia via the PyCall package. I believe the issue lies mostly with PyCall so I have created the following issue there:

https://github.com/JuliaPy/PyCall.jl/issues/735

However, since I'm not getting any answers from PyCall I'm turning to Julia itself. I hope this is no problem.

Basically, converting a Python csc_matrix object to a Julia SparseMatrixCSC object takes ages and uses a lot of RAM. I assume data gets copied, but I have no acces to the source code so I can't confirm.

For more information, you can look at the link above. Any help would be greatly appreciated!

Kind regards,
Tom

**Comment by fredrikekre** [comment]
These type of questions is better to post on the forum: https://discourse.julialang.org/.

**Comment by TomDeWeer** [comment]
Thanks for the fast answer!
